ts/src/base/Exchange.ts

```diff
@@ -3035,6 +3035,7 @@ export default class Exchange {
                         currency['networks'][key]['active'] = false;
                     }
                 }
+                network = currency['networks'][key]; // refresh with possibly new data
```

Comment:
few lines above, network key's "active" value might be rewriten, so we need to "refresh" `network` variable